packages/pigment-css-react/tests/Container/container.test.ts

```diff
@@ -0,0 +1,48 @@
+import path from 'node:path';
```

Comment:
Nit: The test file name should be `Container.test.ts` instead of `container.test.ts` just to follow our convention.